config/always-pam.m4

```diff
@@ -0,0 +1,36 @@
+AC_DEFUN([ZFS_AC_CONFIG_ALWAYS_PAM], [
+	AC_ARG_ENABLE(pam,
+		AC_HELP_STRING([--enable-pam],
+		[install pam_zfs_key module [[default: check]]]),
+		[enable_pam=$enableval],
+		[enable_pam=check])
+
+	AC_ARG_WITH(pammoduledir,
+		AC_HELP_STRING([--with-pammoduledir=DIR],
+		[install pam module in dir [[/lib/security]]]),
+		pammoduledir=$withval,pammoduledir=/lib/security)
```

Comment:
This warrants more quoting for robustness (elsewhere too).
`[pammoduledir="$withval"], [pammoduledir=/lib/security])`